pallets/phala/src/stakepool.rs

```diff
@@ -692,6 +697,29 @@ pub mod pallet {
 		}
 	}
 
+	impl<T: Config> Ledger<T::AccountId, BalanceOf<T>> for Pallet<T> {
+		fn ledger_accrue(who: &T::AccountId, amount: BalanceOf<T>) {
+			let b: BalanceOf<T> = StakeLedger::<T>::get(who).unwrap_or_default();
+			StakeLedger::<T>::insert(who, b.saturating_add(amount));
+			Self::update_lock(who, b.saturating_add(amount));
+		}
+
+		fn ledger_reduce(who: &T::AccountId, amount: BalanceOf<T>) {
+			let b: BalanceOf<T> = StakeLedger::<T>::get(who).unwrap_or_default();
+			let new_b = if b > amount {
+				b.saturating_sub(amount)
+			} else {
+				Zero::zero()
+			};
```

Comment:
```suggestion
			let new_b = b.saturating_sub(amount);
```

Not necessary to compare `b` and `amount` here, because `saturating_` series methods implies to clip the calculation between [0, MAX)